tools/toitlsp/cmd/repro.go

```diff
@@ -139,8 +139,8 @@ func serveRepro(cmd *cobra.Command, args []string) error {
 	documents := lsp.NewDocuments(logger)
 	fs := lsp.MultiFileSystem{lsp.NewDocsCacheFileSystem(documents), reproFS}
 
-	fileServer := compiler.NewFileServer(fs, logger, reproFS.sdkPath)
-	go fileServer.ListenAndServe(fmt.Sprintf(":%d", port))
+	fileServer := compiler.NewPortFileServer(fs, logger, reproFS.sdkPath, fmt.Sprintf(":%d", port))
+	go fileServer.Start()
```

Comment:
I prefer `ListenAndServe` because it mimics the http server API in go, but if you really want to rename this to `Run`. 
Normally`Start` is non-blocking whereas`Run` is blocking.

Reply:
that will make it relatable to `os/exec.Cmd`

Reply:
Sounds good. Thanks.

The reason I'm changing it to `Run` is because I'm adding another server that only uses a pipe.